flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/utils/ExecutorUtils.java

```diff
@@ -79,6 +79,7 @@ public static void setBatchProperties(StreamGraph streamGraph, TableConfig table
 		streamGraph.setAllVerticesInSameSlotSharingGroupByDefault(false);
 		streamGraph.setScheduleMode(ScheduleMode.LAZY_FROM_SOURCES_WITH_BATCH_SLOT_REQUEST);
 		streamGraph.setStateBackend(null);
+		streamGraph.setCheckpointStorage(null);
```

Comment:
@aljoscha I'm pretty certain this is correct but could you please double-check. I'm also not sure if I need to explicitly do something for batch execution somewhere else now that it's supported for DataStream.